recipes/tsil/all/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.4)
+project(TSIL C)
```

Comment:
As much as I love CMake it's very hard to see if this is correct...

I noticed in the docs https://www.niu.edu/spmartin/TSIL/README.txt there is Makefile support.

Did you consider using the Conan helper for that? https://docs.conan.io/en/latest/howtos/makefiles.html. It's usually a safe bet to use upstreams build files

Reply:
Hi @prince-chrismc ,

many thanks for your help with the recipe!

Yes, I've considered using the `Makefile` that is shipped with `TSIL`. Originally I used this `Makefile` in the recipe, see https://github.com/conan-hep/conan-tsil/blob/release/1.44/conanfile.py#L51 . However, I had to add some manual quirks in order to be able to cross-compile TSIL (tested with travis-ci back then) on Linux and Mac. Furthermore, the original `Makefile` is not supported in Visual Studio, as far as I know. Third, the original `Makefile` of TSIL is really trivial, so creating an equivalent `CMakeLists.txt` was not a big deal. :) So, from my perspective it has the advantage that it more platform independent and fits much easier into a conan recipe. :)